tests/languages/java-file/app/MainApp.java

```diff
@@ -0,0 +1,22 @@
+import java.io.*;
+
+public class MainApp
+{
+	public static void main(String[] args) throws IOException {
+		System.out.println("Hello SGX world from MainApp Java!");
+		FileReadSample frs = new FileReadSample();
+		System.out.println("Trying to read file /app/input.txt");
+		String data = frs.readFile("/app/input.txt");
+
+		System.out.println("Content of the file:");
+		System.out.println(data);
+		
+		if (data.compareTo("This is the first line of the file\nThis is the second line of the file\n") == 0) {
```

Comment:
I was thinking of this:
```
	        if (data.compareTo("This is the first line of the file\nThis is the second line of the file\n") != 0) {
		    System.exit(1);
		}
```

Would that not be enough?

Reply:
Sure will do the necessary change.

Reply:
Done